### ISSUE TITLE
There are 2 dropout papers

### ISSUE THREAD

**Issue by liviu-** [terryum/awesome-deep-learning-papers#33]
Hi,

In section [Optimization / Training Techniques](https://github.com/terryum/awesome-deep-learning-papers/blob/master/README.md#optimization--training-techniques), there are: 

- Dropout: A simple way to prevent neural networks from overfitting (2014)
- Improving neural networks by preventing co-adaptation of feature detectors (2012)

Both papers have the same authors, and they both introduce dropout. However, the papers are 2 years apart and the content is not identical. Was the paper just republished after being edited by Bengio, or is there some story behind this? Is it helpful to have both on the page?

**Comment by terryum** [comment]
I think the 2014 paper is a extension of the 2012 paper. 
Dropout is first proposed in the 2012 paper and more theoretical and empirical studies are included in the 2014 paper. Since the both papers have cited more than 1500 times, I think I cannot easily remove one of them from the list...
Thank you for your opinion.
